{"uid":"184e9d4b97b3d2de","name":"Login with valid credentials test","fullName":"tests.test_login.TestLogin#test_valid_login","historyId":"e7f254b9bfd5f2bc8d4a1ffa6dc7aebb","time":{"start":1707008732145,"stop":1707008742337,"duration":10192},"description":"valid login","descriptionHtml":"<p>valid login</p>\n","status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x564718ed8d93 <unknown>\n#1 0x564718bbc337 <unknown>\n#2 0x564718c07d60 <unknown>\n#3 0x564718c07e51 <unknown>\n#4 0x564718c4bbc4 <unknown>\n#5 0x564718c2a46d <unknown>\n#6 0x564718c4912d <unknown>\n#7 0x564718c2a1e3 <unknown>\n#8 0x564718bfa135 <unknown>\n#9 0x564718bfb13e <unknown>\n#10 0x564718e9ce4b <unknown>\n#11 0x564718ea0dfa <unknown>\n#12 0x564718e896d5 <unknown>\n#13 0x564718ea1a6f <unknown>\n#14 0x564718e6d69f <unknown>\n#15 0x564718ec6098 <unknown>\n#16 0x564718ec6262 <unknown>\n#17 0x564718ed7f34 <unknown>\n#18 0x7f6a9ba94ac3 <unknown>","statusTrace":"self = <tests.test_login.TestLogin object at 0x7f562dbce5d0>\nsecret_data = {'email': '', 'password': ''}\njson_data = {'forgot_password': {'error_message': \"We can't find a user with that e-mail address.\", 'success_message': 'A reset li...you page', 'name': 'test', 'new_name': 'another test', 'no_project_found_msg': 'No project matches the criteria', ...}}\n\n    @allure.description(\"valid login\")\n    @allure.title(\"Login with valid credentials test\")\n    @allure.tag(\"Tagged test\")\n    @pytest.mark.run(order=1)\n    def test_valid_login(self, secret_data: dict, json_data: dict):\n>       self.about_page.click_login_link()\n\ntests/test_login.py:39: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/about_page.py:20: in click_login_link\n    self.click(self.LOGIN_LINK)\npages/base_page.py:38: in click\n    el: WebElement = self.wait.until(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"ca9c0da2d5dd9c2507ab1e1fb94ea525\")>\nmethod = <function element_to_be_clickable.<locals>._predicate at 0x7f562dbbc860>\nmessage = ''\n\n    def until(self, method, message: str = \"\"):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x564718ed8d93 <unknown>\nE       #1 0x564718bbc337 <unknown>\nE       #2 0x564718c07d60 <unknown>\nE       #3 0x564718c07e51 <unknown>\nE       #4 0x564718c4bbc4 <unknown>\nE       #5 0x564718c2a46d <unknown>\nE       #6 0x564718c4912d <unknown>\nE       #7 0x564718c2a1e3 <unknown>\nE       #8 0x564718bfa135 <unknown>\nE       #9 0x564718bfb13e <unknown>\nE       #10 0x564718e9ce4b <unknown>\nE       #11 0x564718ea0dfa <unknown>\nE       #12 0x564718e896d5 <unknown>\nE       #13 0x564718ea1a6f <unknown>\nE       #14 0x564718e6d69f <unknown>\nE       #15 0x564718ec6098 <unknown>\nE       #16 0x564718ec6262 <unknown>\nE       #17 0x564718ed7f34 <unknown>\nE       #18 0x7f6a9ba94ac3 <unknown>\n\n.venv/lib/python3.11/site-packages/selenium/webdriver/support/wait.py:95: TimeoutException","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_verify_url","time":{"start":1707008732144,"stop":1707008732144,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"base_url","time":{"start":1707008732144,"stop":1707008732144,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"json_data","time":{"start":1707008732144,"stop":1707008732144,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"secret_data","time":{"start":1707008732144,"stop":1707008732144,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"description":"valid login","status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x564718ed8d93 <unknown>\n#1 0x564718bbc337 <unknown>\n#2 0x564718c07d60 <unknown>\n#3 0x564718c07e51 <unknown>\n#4 0x564718c4bbc4 <unknown>\n#5 0x564718c2a46d <unknown>\n#6 0x564718c4912d <unknown>\n#7 0x564718c2a1e3 <unknown>\n#8 0x564718bfa135 <unknown>\n#9 0x564718bfb13e <unknown>\n#10 0x564718e9ce4b <unknown>\n#11 0x564718ea0dfa <unknown>\n#12 0x564718e896d5 <unknown>\n#13 0x564718ea1a6f <unknown>\n#14 0x564718e6d69f <unknown>\n#15 0x564718ec6098 <unknown>\n#16 0x564718ec6262 <unknown>\n#17 0x564718ed7f34 <unknown>\n#18 0x7f6a9ba94ac3 <unknown>","statusTrace":"self = <tests.test_login.TestLogin object at 0x7f562dbce5d0>\nsecret_data = {'email': '', 'password': ''}\njson_data = {'forgot_password': {'error_message': \"We can't find a user with that e-mail address.\", 'success_message': 'A reset li...you page', 'name': 'test', 'new_name': 'another test', 'no_project_found_msg': 'No project matches the criteria', ...}}\n\n    @allure.description(\"valid login\")\n    @allure.title(\"Login with valid credentials test\")\n    @allure.tag(\"Tagged test\")\n    @pytest.mark.run(order=1)\n    def test_valid_login(self, secret_data: dict, json_data: dict):\n>       self.about_page.click_login_link()\n\ntests/test_login.py:39: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/about_page.py:20: in click_login_link\n    self.click(self.LOGIN_LINK)\npages/base_page.py:38: in click\n    el: WebElement = self.wait.until(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"ca9c0da2d5dd9c2507ab1e1fb94ea525\")>\nmethod = <function element_to_be_clickable.<locals>._predicate at 0x7f562dbbc860>\nmessage = ''\n\n    def until(self, method, message: str = \"\"):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x564718ed8d93 <unknown>\nE       #1 0x564718bbc337 <unknown>\nE       #2 0x564718c07d60 <unknown>\nE       #3 0x564718c07e51 <unknown>\nE       #4 0x564718c4bbc4 <unknown>\nE       #5 0x564718c2a46d <unknown>\nE       #6 0x564718c4912d <unknown>\nE       #7 0x564718c2a1e3 <unknown>\nE       #8 0x564718bfa135 <unknown>\nE       #9 0x564718bfb13e <unknown>\nE       #10 0x564718e9ce4b <unknown>\nE       #11 0x564718ea0dfa <unknown>\nE       #12 0x564718e896d5 <unknown>\nE       #13 0x564718ea1a6f <unknown>\nE       #14 0x564718e6d69f <unknown>\nE       #15 0x564718ec6098 <unknown>\nE       #16 0x564718ec6262 <unknown>\nE       #17 0x564718ed7f34 <unknown>\nE       #18 0x7f6a9ba94ac3 <unknown>\n\n.venv/lib/python3.11/site-packages/selenium/webdriver/support/wait.py:95: TimeoutException","steps":[{"name":"Click Login link","time":{"start":1707008732145,"stop":1707008742336,"duration":10191},"status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x564718ed8d93 <unknown>\n#1 0x564718bbc337 <unknown>\n#2 0x564718c07d60 <unknown>\n#3 0x564718c07e51 <unknown>\n#4 0x564718c4bbc4 <unknown>\n#5 0x564718c2a46d <unknown>\n#6 0x564718c4912d <unknown>\n#7 0x564718c2a1e3 <unknown>\n#8 0x564718bfa135 <unknown>\n#9 0x564718bfb13e <unknown>\n#10 0x564718e9ce4b <unknown>\n#11 0x564718ea0dfa <unknown>\n#12 0x564718e896d5 <unknown>\n#13 0x564718ea1a6f <unknown>\n#14 0x564718e6d69f <unknown>\n#15 0x564718ec6098 <unknown>\n#16 0x564718ec6262 <unknown>\n#17 0x564718ed7f34 <unknown>\n#18 0x7f6a9ba94ac3 <unknown>\n\n","statusTrace":"  File \"/home/runner/work/kisangateway-test/kisangateway-test/.venv/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/kisangateway-test/kisangateway-test/pages/about_page.py\", line 20, in click_login_link\n    self.click(self.LOGIN_LINK)\n  File \"/home/runner/work/kisangateway-test/kisangateway-test/pages/base_page.py\", line 38, in click\n    el: WebElement = self.wait.until(\n                     ^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/kisangateway-test/kisangateway-test/.venv/lib/python3.11/site-packages/selenium/webdriver/support/wait.py\", line 95, in until\n    raise TimeoutException(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[{"uid":"eb9bf47b87d8d20f","name":"public ip address","source":"eb9bf47b87d8d20f.txt","type":"text/plain","size":11},{"uid":"27a21f1b2152a22","name":"Cookies","source":"27a21f1b2152a22.json","type":"application/json","size":2},{"uid":"279e122250c95ed","name":"Session Storage","source":"279e122250c95ed.json","type":"application/json","size":2},{"uid":"e51d094a2469185e","name":"Local Storage","source":"e51d094a2469185e.json","type":"application/json","size":77},{"uid":"64e82bc89b7394d7","name":"Console Logs","source":"64e82bc89b7394d7.json","type":"application/json","size":599},{"uid":"f6a4bb2baf049158","name":"Full Page Screenshot","source":"f6a4bb2baf049158.png","type":"image/png","size":2649073},{"uid":"de918757b1d5ff63","name":"URL","source":"de918757b1d5ff63.uri","type":"text/uri-list","size":25},{"uid":"d9a52c6adfcf86b","name":"Network Logs","source":"d9a52c6adfcf86b.json","type":"application/json","size":1281399}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":1,"attachmentsCount":8,"hasContent":true,"attachmentStep":false},"afterStages":[],"labels":[{"name":"feature","value":"Login"},{"name":"tag","value":"Tagged test"},{"name":"epic","value":"Security"},{"name":"severity","value":"blocker"},{"name":"tag","value":"run(order=1)"},{"name":"tag","value":"security"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_login"},{"name":"subSuite","value":"TestLogin"},{"name":"host","value":"fv-az573-702"},{"name":"thread","value":"2246-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_login"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"blocker","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":["security","Tagged test","run(order=1)"]},"source":"184e9d4b97b3d2de.json","parameterValues":[]}